psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -1194,6 +1194,17 @@ void OEProp::compute_dipole(bool transition) {
     s << title_ << " DIPOLE Z";
     Process::environment.globals[s.str()] = dipole->get(2);
     wfn_->set_scalar_variable(s.str(), dipole->get(2));
+
+    // Dipole array in au
```

Comment:
So `DIPOLE X` etc. are in Debyes and stored in `dipole`, but the `dipole_array` should be in au's, hence the conversion?

Reply:
Yes, the Debye was a bad choice years ago. So for scalar qcvars, dipole & quadrupole are in D and D A, whereas octupole components are in au. Now that array qcvars exist and aren't a pain to use in qcschema (thanks to Daniel's msgpack, pydantic, and shape setup), this converts both the shape and the units.

Possibly I should have edited the called fn https://github.com/psi4/psi4/pull/1800/files#diff-91bf9e90eb569eec33cd0e5edae2e2b4R1352 to instead return au instead of un-debye-ing the values here, but with other modules calling oeprop, I didn't want to risk it.